### PR TITLE
Add promscale info to telemetry example

### PR DESCRIPTION
**Pull request by davidkohn88**

### CODE REVIEW

**Review by svenklemm — APPROVED** [review]

**Review by solugebefola — APPROVED** [review]

File: using-timescaledb/telemetry.md [hunk 1]
```diff
@@ -20,30 +20,33 @@ the JSON that is sent to our servers about a specific deployment:
...
+	"postgresql_version": "12.4",
+	"timescaledb_version": "1.7.0",
 	"build_architecture": "x86_64",
 	"build_architecture_bit_size": "64",
 	"build_os_name": "Linux",
 	"build_os_version": "4.9.125-linuxkit",
 	"data_volume": "65982148",
 	"db_metadata":
-    	{
+		{
```
Comment (mfreed) [inline using-timescaledb/telemetry.md:31]: This indentation / formatting looks off?

File: using-timescaledb/telemetry.md [hunk 2]
```diff
@@ -20,30 +20,33 @@ the JSON that is sent to our servers about a specific deployment:
...
     	},
 	"num_hypertables": "3",
 	"num_continuous_aggs": "0",
 	"num_reorder_policies": "1",
 	"num_drop_chunks_policies": "2",
 	"related_extensions":
     	{
-		"pg_prometheus": "false",
-		"PostGIS": "true"
+			"pg_prometheus": "false",
```
Comment (mfreed) [inline using-timescaledb/telemetry.md:41]: indentation / formatting?

**Review by mfreed — APPROVED** [review]

**Review by cevian — APPROVED** [review]
LGTM except for Mike's comments

**Comment by davidkohn88** [comment]
Okay. I think I fixed that.

**Pull request merged**